test-clients/src/main/java/com/hedera/services/bdd/spec/assertions/AccountInfoAsserts.java

```diff
@@ -163,51 +163,41 @@ public AccountInfoAsserts balance(Function<HapiApiSpec, Function<Long, Optional<
 
 	public AccountInfoAsserts sendThreshold(long amount) {
 		registerProvider((spec, o) -> {
-			Assert.assertEquals("Bad send threshold!",
-					amount, ((AccountInfo)o).getGenerateSendRecordThreshold());
+			Assertions.assertEquals(amount, ((AccountInfo) o).getGenerateSendRecordThreshold(), "Bad send threshold!");
 		});
 		return this;
 	}
 
 	public AccountInfoAsserts receiveThreshold(long amount) {
 		registerProvider((spec, o) -> {
-			Assert.assertEquals("Bad receive threshold!",
-					amount, ((AccountInfo)o).getGenerateReceiveRecordThreshold());
+			Assertions.assertEquals(amount, ((AccountInfo) o).getGenerateReceiveRecordThreshold(),
+					"Bad receive threshold!");
 		});
 		return this;
 	}
 
 	public AccountInfoAsserts expiry(long approxTime, long epsilon) {
 		registerProvider((spec, o) -> {
-			long expiry = ((AccountInfo)o).getExpirationTime().getSeconds();
-			Assert.assertTrue(
-					String.format("Expiry %d not in [%d, %d]!", approxTime, expiry - epsilon, expiry + epsilon),
-					Math.abs(approxTime - expiry) <= epsilon);
+			long expiry = ((AccountInfo) o).getExpirationTime().getSeconds();
+			Assertions.assertTrue(Math.abs(approxTime - expiry) <= epsilon,
+					String.format("Expiry %d not in [%d, %d]!", approxTime, expiry - epsilon, expiry + epsilon));
 		});
 		return this;
 	}
 
 	public AccountInfoAsserts expiry(String registryEntry, long delta) {
 		registerProvider((spec, o) -> {
-			long expiry = ((AccountInfo)o).getExpirationTime().getSeconds();
+			long expiry = ((AccountInfo) o).getExpirationTime().getSeconds();
 			long expected = spec.registry().getAccountInfo(registryEntry).getExpirationTime().getSeconds() + delta;
-			Assert.assertEquals("Bad expiry!", expected, expiry);
+			Assertions.assertEquals(expected, expiry, "Bad expiry!");
 		});
 		return this;
 	}
 
 	public AccountInfoAsserts autoRenew(long period) {
 		registerProvider((spec, o) -> {
-			Assert.assertEquals("Bad auto-renew period!",
-					period, ((AccountInfo)o).getAutoRenewPeriod().getSeconds());
-		});
-		return this;
-	}
-
-	public AccountInfoAsserts totalAssociatedTokens(int n) {
```

Comment:
would still be good assert to have even though we are not using it.

Reply:
I disagree. We should come up with a test scenario to use this test code or just simply remove it as I did. Besides, we have the Git history to retrieve any deleted code if it is too tricky/hard to rewrite.